scripts/update_and_rebuild_petsc.sh

```diff
@@ -97,9 +104,8 @@ if [ -z "$go_fast" ]; then
       --with-cxx-dialect=C++11 \
       --with-fortran-bindings=0 \
       --with-sowing=0 \
-      $* \
-
-   make all
-else
-   make all
+      $*
```

Comment:
Do you mind putting the same thing back? `$* \`

Reply:
Not at all... Just forgot the backslash, but it's there now.